### PR TITLE
Issue 146 coherent units

### PR DESCRIPTION
**Pull request by uscholdm**
Update annotations including examples for the coherent unit classes.

### CODE REVIEW

File: docs/ReleaseNotes.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 gist Release Notes
 =====
 
+### Minor Updates
+
+- update annotations for coherent unit classes, per issue [#146](https://github.com/semanticarts/gist/issues/146)
```
Comment (sa-bpelakh) [inline docs/ReleaseNotes.md:6]:
```suggestion
### Patch Updates

- Update annotations for coherent unit classes, per issue [#146](https://github.com/semanticarts/gist/issues/146).
```
This is a patch, as only annotations are modified.

**Review by sa-bpelakh — CHANGES_REQUESTED** [review]
Minor change in release notes.

I know it's not in the scope of this issue, but is there a reason why `CoherentProductUnit`, currently defined as 

```
    EquivalentTo: 
        gist:ProductUnit
         and (gist:hasMultiplicand only 
            ((gist:BaseUnit or gist:CoherentProductUnit or gist:CoherentRatioUnit)))
         and (gist:hasMultiplier only 
            ((gist:BaseUnit or gist:CoherentProductUnit or gist:CoherentRatioUnit)))
```
cannot be simplified to
```
    EquivalentTo: 
        gist:ProductUnit
         and (gist:hasMultiplicand only gist:CoherentUnit)
         and (gist:hasMultiplier only gist:CoherentUnit)
```
Similarly with `CoherentRatioUnit`.

**Comment by uscholdm** [comment]
> Minor change in release notes.
> 
> I know it's not in the scope of this issue, but is there a reason why `CoherentProductUnit`, currently defined as
> 
> ```
>     EquivalentTo: 
>         gist:ProductUnit
>          and (gist:hasMultiplicand only 
>             ((gist:BaseUnit or gist:CoherentProductUnit or gist:CoherentRatioUnit)))
>          and (gist:hasMultiplier only 
>             ((gist:BaseUnit or gist:CoherentProductUnit or gist:CoherentRatioUnit)))
> ```
> 
> cannot be simplified to
> 
> ```
>     EquivalentTo: 
>         gist:ProductUnit
>          and (gist:hasMultiplicand only gist:CoherentUnit)
>          and (gist:hasMultiplier only gist:CoherentUnit)
> ```
> 
> Similarly with `CoherentRatioUnit`.

I don't understand what that reason could be, the two expressions are logically equivalent, should not matter which goes where. If out of scope here, perhaps create as a separate issue?

**Comment by rjyounes** [comment]
I'm in favor of making a separate issue for Boris's suggestion.

File: docs/ReleaseNotes.md [hunk 2]
```diff
@@ -1,6 +1,10 @@
 gist Release Notes
 =====
 
+### Patch Updates
+
+- update annotations for coherent unit classes, per issue [#146](https://github.com/semanticarts/gist/issues/146)
```
Comment (rjyounes) [inline docs/ReleaseNotes.md:6]: "Updated" (with capital u) rather than "update"

**Review by rjyounes — CHANGES_REQUESTED** [review]
Very minor change to release notes requested. Otherwise looks good.

File: docs/ReleaseNotes.md (hunk 2 above)
Comment (sa-bpelakh) [reply docs/ReleaseNotes.md:6]: And the trailing period I pointed out in my first suggestion :)

**Review by rjyounes — APPROVED** [review]

**Pull request merged**